src/formats/micropolis.textpb

```diff
@@ -1,4 +1,4 @@
-comment: 'Micropolis MetaFloppy 630kB 5.25" DSDD hard-sectored'
+comment: 'Micropolis MetaFloppy 630kB 5.25" DSDD hard-sectored IMG data-only'
```

Comment:
For consistency with the other profiles, I'd prefer to have this be the normal profile (i.e. the VGI profiles are labelled as such, rather than the user-data-only profile being specially labelled).

Reply:
Actually, thinking about this, there's a cleaner way. You can actually specify multiple profiles on the command line. So if by default, the profiles are configured to read user-data only to .img files, it'd then be possible to make an additional `vgi` profile which changes the filename and sector size to 275. You'd use it like:

`fluxengine read micropolis143 vgi -d something.vgi`

That means the same set of profiles can be used for normal `.img` files and the special `.vgi` files by just adding the modifier. Plus, eventually you'll want a special `.vgi` writer (in order to get the sectors in the right order), and this would be able to patch the configuration to add the `vgi` modifier automatically if you try to read from or write to a VGI file. This should simplify things a lot. WDYT?

Reply:
That works reasonably. I don't know if we'll ever have a VGI writer, but even without this works well.